src/govpress-unit/code-review.md

```diff
@@ -0,0 +1,88 @@
+    * [dalmatian/543](https://github.com/dxw/dalmatian/pull/543)
+1. Full instructions on how the reviewer can test the PR locally (or in some other appropriate way), and what results the reviewer can expect. This section should **never be skipped** when making changes to source code. If the PR will change how a site looks, please include screenshots to help to illustrate the expected changes. For example:
+    * [lambethtogeher/160](https://github.com/dxw/lambethtogether/pull/160)
+    * [govpress-product-theme/167](https://github.com/dxw/govpress-product-theme/pull/167)
+
+If your PR interacts with the database or contains any front-end code, special consideration should be given to security concerns. If you are not sure what security considerations you need to be aware of, please ask a Tech Lead before pushing your changes.
+
+## Reviewing PRs
+
+All developers are expected to undertake code review. Unless a PR only includes documentation, or similar, you should _always_ test the changeset locally. If it is not clear how to do this, ask the PR author using the **Request changes** feature in GitHub, or similar.
+
+PRs should not be held up unnecessarily, so if you have questions about the changeset, or small requests, be clear to tell the author if your comments are non-blocking. Your main role is to prevent bugs appearing on live systems, and to minimise the amount of technical debt in our repositories. If your suggested changes do not achieve either of those aims, they should not block a merge.
+
+Identifying technical debt is a somewhat subjective process, and most developers will get better at this with experience. However, there are a few ways to think about code changes that will help. In particular, try to imagine the next engineer who will be reading these changes or the commit messages for them. They might be:
+
+* A developer from another unit who is debugging an outage in the early hours of the morning, with no context at all for the project.
+* A developer on Support who has never worked on the repository but needs to respond to an urgent client request.
+* An engineer debugging a critical error in production, during an incident, either by manual testing or `git bisect`.
+* In all of these cases, the context around the change you are proposing, the reason for making it, and links to Zendesk tickets or external documentation will be crucial to help the next developer.
+* If you are not sure how easy your commit history will be to understand, run something like `git log --color --graph --oneline --decorate` before you push to a Git server, and consider whether the subject lines of  your commit tell a clear and consistent story.
+
+## After merging
+
+### Deploying your changes
+
+Once a PR has been merged, it will be automatically deployed to a live environment. Initially, this will be a staging environment. Usually, the status of that deploy can be checked with `dalmatian service deploy-status -i <infra> -e <env> -s <service>`.
```

Comment:
Will it be obvious to GovPress colleagues how to fill in the gaps here (infra, env and service)?

Reply:
Yes, I think so. We have a lot more in-house infrastructure than D+ so this should be more familiar to GovPress folks